Dockerfile

```diff
@@ -76,6 +76,7 @@ go install -v ./... && \
 mv $GOPATH/bin/sqlflowserver /usr/local/bin && \
 mv $GOPATH/bin/repl /usr/local/bin && \
 cp -r $GOPATH/src/sqlflow.org/sqlflow/python/sqlflow_submitter /miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/ && \
+cd python/couler/ && python setup.py install && cd ../.. && \
```

Comment:
You can use https://www.tldp.org/LDP/abs/html/subshells.html and shorten this new line into 

```bash
(cd python/couler && python setup.py install)
```

Reply:
Thanks. I've followed the comment.